src/component/requests/full-url.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+const {object, bool} = React.PropTypes;
+
+const _shorten = (str, maxLength) => {
+  if (str.length > maxLength) {
+    return str.substr(0, maxLength) + '…';
+  }
+  return str;
+};
+
+const FullUrl = (props) => {
+  const {request, isShortened} = props;
+
+  let maxUrlLength = 1000;
+  if (isShortened) {
+    maxUrlLength = 60;
+  }
+
+  let port = null;
+  if (request.port) {
+    port = <span className="port">:{request.port}</span>;
+  }
+
+  return <div className="complete-url">
+    <span className="protocol">{request.protocol}//</span>
```

Comment:
I... *uh...*

...apparently that was there originally. That's weird. I'm going to go fix that, good eye.